src/main/java/com/unb/tracker/model/Course.java

```diff
@@ -27,7 +27,6 @@
     private List<Seat> seats;
 
     @ManyToOne(fetch = FetchType.EAGER)
-    @JsonIgnore
```

Comment:
why?

Reply:
We had to remove this annotation for the course instructor to be returned by the ajax call. We need the instructor name to be able to redirect to the right course

Reply:
I believe @jacsmith21 added this, do you thing this will break anything else jacob?

Reply:
@dsteen338 It may. We'll just have to test to make sure everything works 👍